<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face Recognition System</string>
  </property>
  <widget class="QLabel" name="videolabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1980</width>
     <height>1080</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>65</y>
     <width>461</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Serif Condensed</family>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
color: rgb(255, 255, 255);
border-width: 1px;
border-style: solid;
border-color: rgb(255,255,255);
margin-top: 2.5ex
}

QGroupBox::title{
color: rgb(255, 255, 255);
subcontrol-origin: margin;
subcontrol-position: top left;
left: 20px;    
margin-left: 0px;
padding: 1px;
position: relative;
}
</string>
   </property>
   <property name="title">
    <string>Registration</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>140</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>155</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */</string>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#RegButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
color:rgba(255, 255, 255, 210);
border-radius:5px;
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
}

QPushButton#RegButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#RegButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
}

QPushButton#RegButton:disabled{
background-color: rgba(85, 98, 112, 150);
color:rgba(255, 255, 255, 100);
}</string>
    </property>
    <property name="text">
     <string>Register Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CaptureButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#CaptureButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
color:rgba(255, 255, 255, 210);
border-radius:5px;
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
}

QPushButton#CaptureButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#CaptureButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
}

QPushButton#CaptureButton:disabled{
background-color: rgba(85, 98, 112, 150);
color:rgba(255, 255, 255, 100);
}</string>
    </property>
    <property name="text">
     <string>Capture Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TrainButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>360</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#TrainButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
color:rgba(255, 255, 255, 210);
border-radius:5px;
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
}

QPushButton#TrainButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#TrainButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
}
QPushButton#TrainButton:disabled{
background-color: rgba(85, 98, 112, 150);
color:rgba(255, 255, 255, 100);
}</string>
    </property>
    <property name="text">
     <string>Perform Training</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>155</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>OID :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userID">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>271</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(255,255,255,1);
color:rgba(255,255,255,1);
padding-bottom:5px;
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
}

QLineEdit:focus{
border-color:#00E7F2 ;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter your OID (eg. A22801)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>271</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(255,255,255,1);
color:rgba(255,255,255,1);
padding-bottom:5px;
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
}

QLineEdit:focus{
border-color:#00E7F2 ;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter your name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userIC">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>271</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(255,255,255,1);
color:rgba(255,255,255,1);
padding-bottom:5px;
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */
}

QLineEdit:focus{
border-color:#00E7F2 ;
}</string>
    </property>
    <property name="placeholderText">
     <string>for exp: 950810047231</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>200</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>155</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */</string>
    </property>
    <property name="text">
     <string>IC Number :</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="messlabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>1161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>20</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 20pt &quot;Segoe UI Semibold&quot;; 
/* font: 63 16pt &quot;Segoe UI Semibold&quot;; */</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#BackButton{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(51, 122, 183, 1), stop:1 rgba(51, 122, 183, 1));
color:rgba(255, 255, 255, 210);
border-radius:5px;
}

QPushButton#BackButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(150, 123, 111, 219), stop:1 rgba(85, 81, 84, 226));
}

QPushButton#BackButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(150, 123, 111, 255);
}

QPushButton#BackButton:disabled{
background-color: rgba(51, 122, 183, 180);
color:rgba(255, 255, 255, 180);
}</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>560</y>
     <width>641</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	boarder-radius: 10px;
	
}</string>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>680</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/mainBackground.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>680</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:1, stop:1 rgba(0, 0, 0, 205));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>videolabel</zorder>
  <zorder>groupBox</zorder>
  <zorder>messlabel</zorder>
  <zorder>BackButton</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
